zip-0227.rst

```diff
@@ -1,8 +1,359 @@
+1 bit             finalize           boolean                    The boolean that determines the finality of the issuance of that asset
+================= ================== ========================== ========================================================================
+
+We note that the output note commitment of the recipient's notes are not included in the actual transaction, but when added to the global state of the chain, they will be added to the `NoteCommitmentTree` as a shielded note. This prevents future usage of the note from being linked to the issuance transaction, as the nullifier key is not known to the validators and chain observers.
+
+Issuance Bundle
+---------------
+
+An issuance bundle, `IssueBundle`, is the aggregate of all the issuance-related information. Specifically, contains all the issuance actions and the issuer signature on the transaction SIGHASH that validates the issuance itself. It contains the following fields:
+
+- `ik`: the `issuanceValidatingKey`, that allows the validators to verify that the `AssetId` is properly associated with the issuer
+- `actions`: an array of issuance actions, of type `IssueAction`
+- `authorization`: the signature of the transaction SIGHASH, signed by the `issuanceAuthorizingKey`, `isk`, that validates the issuance 
+
+The issuance bundle is then added within the transaction format as a new bundle. This structure will be added to the v6 transaction format.
+
+======= ================== ========================== =========================================================================
+Bytes              Name               Data Type                  Description
+======= ================== ========================== =========================================================================
+Varies  nIssueActions      compactSize                The number of issuance actions in the bundle
+Varies  vIssueActions      IssueAction[nIssueActions] A sequence of issuance actions descriptions
+32      ik                 byte[32]                   The public verification key of the issuer, used to validate the signature
+64      authorization      byte[64]                   The signature of the transaction SIGHASH, signed by the issuer
+======= ================== ========================== =========================================================================
+
+Issuance Protocol
+-----------------
+The issuer program performs the following operations
+
+For all actions `IssueAction`:
+- encode `asset_desc` as 512 byte-string
+- compute `AssetId` as :math:`GroupHash^{\mathbb{P}}(ik || asset_desc)`
+- set the `finalize` boolean as desired (if more more issuance actions are to be created for this asset identifier, set `finalize = 0`, otherwise set `finalize = 1`)
+- For each recipient `i`:
+    - generate a ZSA output note as :math:`\mathsf{ note_i =(d_i, {pk_d}_i, v_i, \rho_i, \psi_i, \mathsf{AssetId}, rcm_i)}`
```

Comment:
```suggestion
    - generate a ZSA output note that includes the Asset Base. For an Orchard-based ZSA note this is :math:`\mathsf{note}_i = (\mathsf{d}_i, \mathsf{pk}_{\mathsf{d},i}, \mathsf{v}_i, \rho_i, \psi_i, \mathsf{AssetBase^{Orchard}}, \mathsf{rcm}_i)}\!`.
```

Reply:
The Asset Base term is undefined, do you mean Asset identifier?
```suggestion
    - generate a ZSA output note that includes the Asset Base. For an Orchard-based ZSA note this is :math:`\mathsf{note}_i = (\mathsf{d}_i, \mathsf{pk}_{\mathsf{d},i}, \mathsf{v}_i, \rho_i, \psi_i, \mathsf{AssetId}, \mathsf{rcm}_i)}\!`.
```

Reply:
Have made this change. The Asset Digest definition will modify this, and I will add a further comment when that is done.

Reply:
Have added the Asset Digest material where it seemed relevant (it wasn't necessary at this location)